src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -180,16 +184,21 @@ const LaunchAnalysisModal = ({
           ['How much does my workflow cost?', icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })]
         ),
       ]),
+
       div({ style: { marginTop: '0.25rem' } }, [
-        h(
-          Link,
-          {
-            style: { verticalAlign: 'top' },
-            href: 'https://support.terra.bio/hc/en-us/articles/360057589931',
-            ...Utils.newTabLinkProps,
-          },
-          ['Set up budget alert', icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })]
-        ),
+        ul({ style: { paddingLeft: '1.5rem' } }, [
+          li([`You are launching ${entityCount} workflow `, entityCount === 1 ? 'run' : 'runs', ' in this submission.']),
+          isFeaturePreviewEnabled(PREVIEW_COST_CAPPING) &&
+            (perWorkflowCostCap !== ''
+              ? li([
+                  `You set a cost limit of ${Utils.formatUSD(perWorkflowCostCap)} per workflow run`,
+                  h('br'),
+                  `x ${entityCount} workflow runs = ${Utils.formatUSD(entityCount * perWorkflowCostCap)}`,
+                  b(' max possible'),
```

Comment:
I feel like `max possible` is too strong. The algorithm attempts to terminate workflows once the limit is reached. Workflows in this situation will spend _more_ than the limit.

Reply:
Blame @admull for terminology!

Reply:
This is good feedback. I liked the suggestion of "approximate maximum" instead of "max possible", so let's change this to "approximate maximum submission cost"